Rakefile

```diff
@@ -1,3 +1,4 @@
+# frozen_string_literal: true
```

Comment:
> Can we pls have a newline and here and for all other files below? I really think this would increase readability.

These changes are all autogenerated by RuboCop, so I'd rather not :smile:.

> Also, can we make this a separate commit?

Sure.